dvc/repo/experiments/utils.py

```diff
@@ -327,7 +327,9 @@ def to_studio_params(dvc_params):
         "params.yaml": {"foo": 1}
     }
     """
-    result = {}
+    result: dict = {}
```

Comment:
3.8 does not support these type hints, unless we import using `from __future__ import annotations`. 3.9 however does. Although I am not sure why mypy does not complain (probably due to type stubs).

```suggestion
    result: Dict = {}
```